### DISCUSSION TITLE
Want to generate prisma schemas during `keystone prisma ...` commands

### DISCUSSION THREAD

**Discussion by pahaz** [keystonejs/keystone#8697] [Feature requests]
So, I've changed some code and I want to regenerate prisma migrations by `keystone prisma migrate dev --create-only`.

At the moment I have got an error `Your Prisma and GraphQL schemas are not up to date`.

As I can see here: https://github.com/keystonejs/keystone/blob/main/packages/core/src/scripts/prisma.ts#L23 
<img width="877" alt="image" src="https://github.com/keystonejs/keystone/assets/1394025/a9720511-e5b3-4b9f-a1e5-0a99a1dee767">

It's better to call `generateTypescriptTypesAndPrisma` before the validation. And, probably we can remove `validatePrismaAndGraphQLSchemas` because it's ok if we have the wrong GraphQL schemas.

**Comment by pahaz** [comment]
https://github.com/keystonejs/keystone/pull/8698

**Reply by dcousens** [reply]
Relevant comment https://github.com/keystonejs/keystone/pull/8698/files#r1266099112

**Comment by pahaz** [comment]
Relly it's not enough to just call `generateTypescriptTypesAndPrisma` we need to call `generatePrismaAndGraphQLSchemas`.

This solution is not good for read-only containers. If there are no changes, there is no need to update any files; therefore, the correct approach is to verify the modifications.

**Reply by pahaz** [reply]
But at the moment it's better for DX and you don't need to run the `keystone dev` command before the migrations.
